config/src/config.rs

```diff
@@ -324,8 +323,12 @@ pub struct Mining {
     pub data_request_timeout: Duration,
     /// Genesis block path
     pub genesis_path: String,
-    /// Binary flag to create a mint with a split reward
-    pub split_mint: bool,
+    /// Percentage to redistribute mint reward in another address
```

Comment:
I would prefer to not implement this feature yet, open a different PR for that

Reply:
+1

Reply:
We discussed before that it could be reviewed together

Reply:
...as long as it's in different commits :rofl: